### ISSUE TITLE
fix: generating a Windows applications displays 2022 copyright

### ISSUE THREAD

**Issue by alestiago** [VeryGoodOpenSource/very_good_templates#162]
**Description**

When creating a Windows application, the copyright notice is 2022 instead of the current year.

**Steps To Reproduce**

1. `mason get`
2. `mason make very_good_core`
3. See `my_app/windows/runner/Runner.rc`:

```txt
VALUE "LegalCopyright", "Copyright (C) 2022 com.example. All rights reserved." "\0"
```

**Expected Behavior**
Legal copyright should be updated to current year when generated.

**Additional Context**

- Can be solved by providing a Mustache variable "current_year" and templating it in.

**Comment by Antoniobox** [comment]
Hello,
Could you assign me to this issue? I’d like to work on it

**Comment by alestiago** [comment]
@Antoniobox solved the issue for Very Good Core with #188, I believe we're missing the same change for Flame Game before closing.

**Comment by alestiago** [comment]
Closing this as completed, thanks @Antoniobox for the efforts!
